### DISCUSSION TITLE
Send configuration through email

### DISCUSSION THREAD

**Discussion by donaldzou** [WGDashboard/WGDashboard#547] [Polls]
Hi all,

I've been trying to figure this out, I first thought is gonna be an easy task but it turns out to be a difficult decision. I listed some pros and cons from my research

**Solution 1 - Email Sending Service**
- Pros:
    1. It's easy to use, got a UI to configure your email template
    2. It is send from a verified source (i.e won't go into spam folder in most case)
    3. Got an API to use
- Cons:
    1. You'll need to get a domain
    2. Not all platform provide free quota
    3. Some platform might not available in your region

**Solution 2 - Plain old SMTP**
- Pros:
    1. It is available is most email provider, you'll just need an email account
    2. It works with self hosted email as well
    3. Is most case, it is **FREE**
- Cons:
    1. **According to me research, Outlook and Gmail no longer provide SMTP server for their free account, which means you can't login to it with your passoword, I'm not sure with other providers.**

**Comment by norekluzt** [comment]
With gmail you get to create app passwords for this specific use case. ive tested it and it works like a charm. Thank you for this feature.
